examples/main.tf

```diff
@@ -2,9 +2,8 @@ provider "aws" {
   region = "eu-west-1"
 }
 
-
 locals {
-  name            = "s3-bucket-${uuid()}"
+  name            = "s3-bucket"
```

Comment:
S3 names are unique for all clients, here you can use this weird uuid since for sure `s3-bucket` will not work, plus I'm 100% sure you didn't test it successfully :/